src/lib/components/Search.svelte

```diff
@@ -0,0 +1,76 @@
+<script lang="ts">
+	export let search: string;
+	export let placeholder = 'Search...';
+	export let label: string | null = null;
+	export let name = `search_${Math.random().toString(36).substring(7)}`;
+</script>
+
+<div class="group">
+	{#if label}
+		<label for={name}>{label}</label>
+	{/if}
+	<div class="input">
+		<input bind:value={search} type="search" {name} {placeholder} />
+
+		<button class="submit" on:click>
+			<svg
```

Comment:
This is just a comment.

It will be easier in the future to have a SVG collection somewhere, and create an `<Icon />` component.